### ISSUE TITLE
make OCR add-on ready for NVDA 2019.1 by providing apropriate values in manifest for add-on compatibility.

### ISSUE THREAD

**Issue by lukaszgo1** [nvaccess/nvda-ocr#5]
The OCR add-on is for many users of Windows versions older than 10 one of the most important plugins for NVDA. It should be updated to provide compatibility with add-on compatibility check introduced in today's snapshot. Furthermore this add-on is developed by NVAcces and they should give an example of conforming to the new requirements.

**Comment by lukaszgo1** [comment]
#6 is basically the same request, so closing.
